data/RWTH-NFDI-Jupyter-Service.yaml

```diff
@@ -0,0 +1,42 @@
+title: RWTH Aachen University
+provider: HPC.ITC.RWTH
+service_url: https://jupyterhub.hpc.itc.rwth-aachen.de:9651/
+support: servicedesk@itc.rwth-aachen.de
+health_api_url: https://jupyterhub.hpc.itc.rwth-aachen.de:9651/hub/api/health
+documentation_url: https://help.itc.rwth-aachen.de/service/rhr4fjjutttf/article/689934fec5a34c909c54606f6bc2e827/
+target_group_open_for: Researchers, RWTH Students/Employees/Guests, Juelich Members, Users sponsored by other Users, Users upon project application 
+login_process: Login with RWTH HPC Account through most NHR or NRW VPNs
+features:
+  version: JupyterHub 3.0+, JupyterLab 3.4.7+
+  kernels: ["Python 3.9", "Octave", "R", "C++ 11/14/17", "Matlab 2022b", "Matlab 2020b",  "Python: Keras", "Python: Tensorflow",  "Python: Tensorboard", "Python: Gurobi", "Python: Pytorch" ]
+  extensions: ["ipytree",
+        "jupyterlab-drawio",
+        "jupyterlab-plotly",
+        "jupyterlab_iframe",
+        "nbdime-jupyterlab",
+        "jupyter-matplotlib",
+        "jupyterlab_pygments",
+        "jupyterlab/git",
+       " jupyterlab/latex",
+        "jupyter-widgets/jupyterlab-manager"]
+  proxy_apps: ["Matlab 2022b", "Matlab 2020b"]
+  install: false
+  shared_folder: true
+  persistent_storage: true
+  misc: ["Available core-hours vary based on user affiliation or project.", "Runs on HPC Hardware", "Users installs planned, by request only at the moment."]
```

Comment:
`Available core-hours vary based on user affiliation or project.`
Is this reflected in the range of **_cpu_time** (4-172 h; lines 33/34 below)? If yes, it can be skipped here. If there are other differences (especially in usable kernels / apps) it should be rephrased.

Reply:
It is not related. The core-hours available over the lifetime of the computing time is different and important to note.

Reply:
Do you mean the difference between core hours (the time the cpu really does something) and the wall time (time from start to end of the slurm job = jupyter server)? If yes, what of the two did you specify here under misc? The value under 'resources' should refer to core hours (cpu time).